main.go

```diff
@@ -1,3 +1,5 @@
+//go:generate go-winres.exe make
```

Comment:
Have you considered using `//go:generate go-winres make --product-version=git-tag`  to build the version directly from the latest git tag?

Reply:
updated the PR to use this and removed the json file

Reply:
had to remove this, and added the versioning in the `Makefile` since we'll have to upload the source tarball and that doesn't have git history it fails while building on brew, i have updated the (https://github.com/crc-org/win32-background-launcher/pull/1/commits/61234b7ec1affb70cc2e6e87ba1149e700864c5e) in the PR again!